specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/ManagedDatabaseMoveOperations.json

```diff
@@ -0,0 +1,293 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/locations/{locationName}/managedDatabaseMoveOperationResults": {
+      "get": {
+        "tags": [
+          "ManagedDatabaseMoveOperations"
+        ],
+        "description": "Gets managed database move operations.",
+        "operationId": "ManagedDatabaseMoveOperations_ListByLocation",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=37&length=2)

use v3 types

Reply:
This JSON is automatically generated, so I'd prefer to have the Swagger generation script modified rather than to do manual intervention. Would this work?